WBooks/HomeViewController.swift

```diff
@@ -25,7 +25,13 @@ final class HomeViewController: UITabBarController {
         static let addNewSelectedImage = UIImage(named: "ic_add_new_active")
         static let rentalsName = "Rentals"
         static let rentalsImage = UIImage(named: "ic_rentals")
-        static let rentalsSelectedImage = UIImage(named: "ic_rentals_active")
+        static let rentalsSelectedImage = UIImage(named: "ic_rentals_active") 
+        static let navigationBarImage = UIImage(named: "bc_nav bar")
+        static let statusBarOriginX: CGFloat = 0
+        static let statusBarOriginY: CGFloat = 0
```

Comment:
You can do `static let statusBarOrigin: CGPoint = CGPoint(x: 0, y: 0)`.

But why do you need this? won't it always be in origin 0 because we won't it on top?
Remember status bar and navigation bar are different things